chai/chai.d.ts

```diff
@@ -25,6 +28,28 @@ declare module Chai {
         (target: any, message?: string): Assertion;
     }
 
+    interface ShouldAssertion {
+        equal(val1: any, val2: any, msg?: string): void;
+        Throw: ShouldThrow;
+        throw: ShouldThrow;
+        exist(val: any, msg?: string): void;
+    }
+
+    interface Should extends ShouldAssertion {
+        not: ShouldAssertion;
+        fail(actual: any, expected: any, message?: string, operator?: string): void;
+    }
+
+    interface ShouldThrow {
+        (actual: Function): void;
+        (actual: Function, expected: string, message?: string): void;
```

Comment:
You can use a union type here to remove the line below it by defining `expected: string|RegExp`. I'm sure there are other opportunities for this in this file as well, including code you didn't write. This is a fairly fresh feature, so it doesn't surprise me that it doesn't currently show in this file, but it's definitely something we should be doing from here on out.